src/components/CameraControls/index.tsx

```diff
@@ -169,64 +215,75 @@ const CameraControls = ({
                         </Button>
                     </Tooltip>
                 </div>
-                <Tooltip
-                    placement="left"
-                    title="Focus (F)"
-                    color={TOOLTIP_COLOR}
-                >
-                    <Button
-                        size="small"
-                        className={classNames([
-                            { [styles.active]: isFocused },
-                            styles.radioBtn,
-                        ])}
-                        onClick={() => saveFocusMode(!isFocused)}
+            </div>
+            <div className={styles.moveButtons}>
+                <div className={styles.radioGroup}>
+                    <Tooltip
+                        placement="left"
+                        title={
+                            mode === ROTATE
+                                ? "Orthographic Camera"
```

Comment:
what's the reason for this ternary? it's giving the same title in each case?

Reply:
that's a mistake, fixed